src/components/EntityInfo.vue

```diff
@@ -162,7 +172,7 @@ export default class EntityInfo extends Mixins(CommonMixin, EnumMixin) {
 
   /** True if View and Change Company Info button should be rendered. */
   private get viewChangeInfoEnabled (): boolean {
-    return (this.isBComp || this.isBcCompany || this.isUlc) && getFeatureFlag('alteration-ui-enabled')
```

Comment:
This flag is now True in all environments, so I deleted it (from this project only -- it is still used (and possibly useful) in the Edit UI).